js/app/dashboard/store/reducers.js

```diff
@@ -251,12 +253,54 @@ const taskListGroupMode = (state = 'GROUP_MODE_FOLDERS', action) => {
   }
 }
 
+const tasksFilters = (state = {showFinishedTasks: true}, action) => {
+  switch (action.type) {
+    case 'TOGGLE_SHOW_FINISHED_TASKS':
+      let showFinishedTasks = !state.showFinishedTasks
+      return {...state, showFinishedTasks}
+    default:
+      return state
+  }
+}
+
+const allTags = (state = tagsInitial, action) => {
```

Comment:
Useless - as you don't react to any redux events, it appears that you don't need to introduce this reducer